core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -75,19 +75,18 @@ public static AlluxioBlockStore create() {
    * @return the {@link AlluxioBlockStore} created
    */
   public static AlluxioBlockStore create(FileSystemContext context) {
-    return new AlluxioBlockStore(context, NetworkAddressUtils.getClientHostName());
+    return new AlluxioBlockStore(context, TieredIdentityFactory.getInstance());
   }
 
   /**
    * Creates an Alluxio block store.
    *
    * @param context the file system context
-   * @param localHostName the local hostname for the block store
+   * @param tieredIdentity the tiered identity
    */
-  public AlluxioBlockStore(FileSystemContext context, String localHostName) {
+  public AlluxioBlockStore(FileSystemContext context, TieredIdentity tieredIdentity) {
```

Comment:
Is this for testing only, the tiered identity should be fixed

Reply:
changed visibility